odk/odk.py

```diff
@@ -552,6 +552,10 @@ class OntologyProject(JsonSchemaMixin):
     uribase : str = "http://purl.obolibrary.org/obo"
     """Base URI for PURLs. DO NOT MODIFY AT THIS TIME, code is still hardwired for OBO """
```

Comment:
I guess this should be updated? `uribase` *can* be modified now, that’s even the entire point of this PR? Or do we want to allow modifying it while still strongly discouraging the practice?

Reply:
Updated documentation. No I dont want to discourage it. I want to widen the scope of ODK so that non OBO ontologies are supported the same way as OBO ones!